### ISSUE TITLE
sub training set

### ISSUE THREAD

**Issue by argosdh** [hustvl/HAIS#5]
Hi, thx for your great work! I'm trying to use your code and I don't have that much time for training. I'd like to make a smaller sub training & validation set for algorithm checking. Do you have any suggestions or did you use any sub sets for quick checking? 

Here is what I did:
I randomly choose 300 scans out of 1201 scans, zoomed out _training epochs_(500 -> 125) and other epochs like _preparing epoch_(100 -> 25) and _cal_iou_based_on_mask_start_epoch_(200 -> 50).  I still used the 312 val set for validation. The val result is absolutely very poor. AP/AP50/AP25 is like: 0.005/0.014/0.043. I think there should be a better setting about sub set choosing and config parameters setting for a quick check. Or you need several days of training for checking a new modification.

**Comment by outsidercsy** [comment]
Hi, thanks for the attention to our work. I think it's not proper to subsample the training set. The diversity of data will affect the performance a lot. For saving training time, I suggest to load the weight of backbone and finetune for a few epochs, which is a common practice in 2D tasks.
